docs/installation.rst

```diff
@@ -1,597 +1,27 @@
 Installation
```

Comment:
```suggestion
:orphan:

Installation
```
Silences the ["document isn't included in any toctree"](https://github.com/python-pillow/Pillow/actions/runs/8107308747/job/22158625064#step:8:147) warning.